fury/gltf.py

```diff
@@ -77,6 +77,8 @@ def actors(self):
             if self.materials[i] is not None:
                 base_col_tex = self.materials[i]['baseColorTexture']
                 actor.SetTexture(base_col_tex)
+                base_color = self.materials[i]['baseColor']
+                actor.GetProperty().SetColor(tuple(base_color[:3]))
```

Comment:
do we have a function in `fury.utils` for doing that?

Reply:
No, we have a similar function `set_polydata_colors`